src/browser-lib/hubspot/forms/getHubspotFormPayloads.ts

```diff
@@ -113,3 +113,93 @@ export const getHubspotDownloadsFormPayload = (props: {
 
   return payload;
 };
+
+type OnboardingUKTeacherProps = {
+  school: string;
+  schoolName: string;
+};
+const isOnboardingUKTeacherProps = (
+  u: unknown,
+): u is OnboardingUKTeacherProps => {
+  return (
+    typeof u === "object" &&
+    u !== null &&
+    "school" in u &&
+    "schoolName" in u &&
+    typeof (u as OnboardingUKTeacherProps).school === "string" &&
+    typeof (u as OnboardingUKTeacherProps).schoolName === "string"
+  );
```

Comment:
Could this be done more declaratively with Zod and safeParse?

Reply:
yesss now why didn't I think of that 😅